tailwind.config.mjs

```diff
@@ -75,6 +75,13 @@ export default {
         'px-tiny': ['10px'],
         'px-base': ['16px'],
       },
+      gradientColorStops: {
+        // These gradient stops define a custom gradient shown at the bottom of
+        // long annotation body excerpts.
+        excerptStop1: 'rgba(255, 255, 255, 0) 50%',
+        excerptStop2: 'rgba(0, 0, 0, 0.08) 95%',
+        excerptStop3: 'rgba(0, 0, 0, 0.13) 100%',
+      },
```

Comment:
I don't imagine we'll reuse this specific gradient, so I would have preferred to use Tailwind's ["arbitrary" utility classname syntax](https://tailwindcss.com/docs/adding-custom-styles#using-arbitrary-values), and not abstract this out. i.e. I would have preferred to use classnames like `from-[arbitrary-gradient-stop-value] to-[arbitrary-gradient-stop-value]`. This would work if our gradient color stops included colors only, but Tailwind does not appear to support arbitrary syntax with gradient stop _positions_ (e.g. `95%`), so for now this is the route I've taken.  It might be possible to make this work inline with the right combination of classname munging (replacing spaces with underscores, etc.) but doesn't seem worth the time just yet.

At this point in the transition process, I'm OK with leaving a few soft edges for review later. My current plan is to get most or all of the sidebar components transitioned and then grep components for arbitrary syntax class names (with square brackets) and look at the resulting Tailwind configuration to spot patterns and whatnot. etc. etc. But not try to be too rigorously sterile right now in a way that will slow this process down (and probably make some assumptions that end up not being true).